list.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative 'my_enumerable'
+
+# Created a new class called List that inherits from MyEnumerable
+class List
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
```

Comment:
- [ ] OPTIONAL
Kindly note that your class is named as `List` instead of `MyList`. Therefore kindly modify your class name to `MyList` or modify the initialized class name accordingly to enable your program run with no errors.